Modules/_sqlite/connection.c

```diff
@@ -1716,22 +1709,18 @@ pysqlite_connection_create_collation(pysqlite_Connection* self, PyObject* args)
     Py_XDECREF(uppercase_name);
 
     if (PyErr_Occurred()) {
-        retval = NULL;
+        return NULL;
     } else {
-        Py_INCREF(Py_None);
-        retval = Py_None;
+        return Py_NewRef(Py_None);
     }
```

Comment:
I would prefer something like:

```
    if (PyErr_Occurred()) {
        return NULL;
    }

    return Py_NewRef(Py_None);
```

To better highlight that the first case is an error case, and the last line is the regular code path.

Reply:
I agree

Reply:
Fixed in dd7d4e797b.